doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -70,57 +72,60 @@ Hot Reload features vary between platforms and IDE, you can check below the list
             <th colspan="2">Rider</th>
         </tr>
         <tr>
-            <th>Target</th>
-            <th>C#</th>
-            <th>XAML</th>
-            <th>C#</th>
-            <th>XAML</th>
-            <th>C#</th>
-            <th>XAML</th>
+            <th>🐞</th>
+            <th>With</th>
+            <th>Without</th>
+            <th>With</th>
+            <th>Without</th>
+            <th>With</th>
+            <th>Without</th>
         </tr>
     </thead>
     <tbody>
         <tr>
             <td>Desktop Windows</td>
-            <td>✅🐞</td><td>✅🐞</td>
-            <td>✅🐞</td><td>✅🐞</td>
             <td>✅</td><td>✅</td>
+            <td>🔳</td><td>✅</td>
+            <td>🔳</td><td>✅</td>
         </tr>
         <tr>
             <td>Desktop/WSL</td>
-            <td>⌛<sup><a href="#hr-footnotes">[1]</a></sup> </td><td>⌛<sup><a href="#hr-footnotes">[1]</a></sup></td>
-            <td>✅</td><td>✅</td>
+            <td>✅</td>
+            <td>✅</td>
+            <td>🔳</td><td>✅</td>
             <td>🔳<sup><a href="#hr-footnotes">[2]</a></sup></td><td>🔳<sup><a href="#hr-footnotes">[2]</a></sup></td>
         </tr>
         <tr>
             <td>iOS</td>
-            <td>⌛<sup><a href="#hr-footnotes">[3]</a></sup></td><td>✅<sup><a href="#hr-footnotes">[4]</a></sup></td>
-            <td>⌛<sup><a href="#hr-footnotes">[3]</a></sup>🛜</td><td>✅<sup><a href="#hr-footnotes">[4]</a></sup>🛜</td>
-            <td>🔳</td><td>🔳</td>
+            <td>✅<sup><a href="#hr-footnotes">[4]</a></sup></td>
+            <td>🔳</td>
```

Comment:
We should have another icon for this case: it's not that this target is not supported, it's only that HR does not work in that configuration.

BTW please keep the alignment of 2 table columns per code code row so one code row is still one "main case" while the two table columns are only with or without debugger

Reply:
The new column is confusing. What was the issue with the debugger icon in the cell?

Reply:
@jeromelaban shared with @Kunal22shah that it would be better to simplify the previous matrix based on the latest changes, and he gave him an example to follow with the debugger with/without columns. I will let you sync with @dr1rrb in regards to these changes.

Reply:
IMO the issue is that the 🐞 indicates the support with the debugger, but it does not indicate that we **cannot** use HR **without** debugger (like mobile in VS). So as the C# vs XAML columns are no longer relevant, I would replace those with a debugger column which clearly indicates if it's supported or not.